src/js/edu-benefits/definitions/veteranId.js

```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash/fp';
+
+import * as ssn from '../../common/schemaform/definitions/ssn';
+
+// TODO: move vaFileNumber out of 1995 and into a shared backend definition
```

Comment:
We could, but in this case the schemas will be identical across all 3 forms, so it seems we should just use a common vaFileNumber definition from the backend.